tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,332 @@
-# Read and plot a segment of raw data
-
-start, stop = raw.time_as_index([100, 115])  # 100 s to 115 s data segment
-data, times = raw[:, start:stop]
-print(data.shape)
-print(times.shape)
-data, times = raw[2:20:3, start:stop]  # access underlying data
-raw.plot()
-
-##############################################################################
-# Save a segment of 150s of raw data (MEG only):
-
-raw.save('sample_audvis_meg_raw.fif', tmin=0, tmax=150, picks='meg',
-         overwrite=True)
-
-##############################################################################
-# Define and read epochs
-# ^^^^^^^^^^^^^^^^^^^^^^
+###############################################################################
+# .. note::
 #
-# First extract events:
-
-events = mne.find_events(raw, stim_channel='STI 014')
-print(events[:5])
-
-##############################################################################
-# Note that, by default, we use stim_channel='STI 014'. If you have a different
-# system (e.g., a newer system that uses channel 'STI101' by default), you can
-# use the following to set the default stim channel to use for finding events:
-
-mne.set_config('MNE_STIM_CHANNEL', 'STI101', set_env=True)
-
-##############################################################################
-# Events are stored as a 2D numpy array where the first column is the time
-# instant and the last one is the event number. It is therefore easy to
-# manipulate.
+#     The ``info`` attribute is :class:`a dictionary-like object <mne.Info>`
+#     that keeps track of channel locations, applied filters, projectors, etc.
+#     and is preserved across :class:`~mne.io.Raw`, :class:`~mne.Epochs`, and
+#     :class:`~mne.Evoked` objects. Notice also the ``chs`` entry, showing that
+#     MNE-Python detects different sensor types and handles each appropriately.
 #
-# Define epochs parameters:
-
-event_id = dict(aud_l=1, aud_r=2)  # event trigger and conditions
-tmin = -0.2  # start of each epoch (200ms before the trigger)
-tmax = 0.5  # end of each epoch (500ms after the trigger)
-
-##############################################################################
-# Exclude some channels (original bads + 2 more):
-
-raw.info['bads'] += ['MEG 2443', 'EEG 053']
-
-##############################################################################
-# The variable ``raw.info['bads']`` is just a Python list.
+# :class:`~mne.io.Raw` objects also have several built-in plotting methods;
+# here we show the power spectral density for each sensor type with
+# :meth:`~mne.io.Raw.plot_psd`, as well as a plot of the raw sensor traces with
+# :meth:`~mne.io.Raw.plot`. In interactive Python sessions,
+# :meth:`~mne.io.Raw.plot` is interactive and allows scrolling, scaling, bad
+# channel marking, annotation, projector toggling, etc.
+
+raw.plot_psd()
+raw.plot(duration=5, n_channels=30)
+
+###############################################################################
+# Preprocessing
```

Comment:
At the end of this section, I would point to a tutorial/example about: 1) ICA 2) Maxfilter 3) SSP 4) Annotations and rejection thresholds

Reply:
I'm actually +0 for adding too many links here (or at end of every section, as you suggested elsewhere). Reasons:

1. The point of this tutorial is that it's (reasonably) quick to go through, and gets everyone on the same page w/r/t some basic data structures and terminology.  The more we link out from it, the more chances are that people won't go all the way through it.  

2. Part of the point of sorting the tutorials into categories ([like here](https://13028-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/index.html)) is that it will be easier to find what you're looking for.

3. Since some/many of these tutorials are slated for getting updated / split / merged / eliminated, adding links now may just get changed / removed in a few weeks.